deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml

```diff
@@ -58,17 +48,6 @@ rules:
   verbs:
   - list
 ### END
-### Start - Allow viewing platform and other infra data in console
-# https://issues.redhat.com/browse/OSD-4298
-- apiGroups:
-  - config.openshift.io
-  resources:
-  - infrastructures
```

Comment:
Removing `infrastructures` as it will be covered under (line 168):

```
- apiGroups:
  - config.openshift.io
  resources:
  - '*'
  verbs:
  - get
  - list
  - watch
```